### PR TITLE
add enableAltField option to boilerplate

### PR DESCRIPTION
**Pull request by ValJed**
Add the option to add an alt field to images, and to use it by default in alt attribute of `apostrophe-images-widgets`.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -13,14 +13,18 @@ var apos = require('apostrophe')({
...
     // least have a minimal configuration here: `moduleName: {}`
 
     // If a template is not found somewhere else, serve it from the top-level
     // `views/` folder of the project
+    'apostrophe-templates': { viewsFolderFallback: path.join(__dirname, 'views') },
 
-    'apostrophe-templates': { viewsFolderFallback: path.join(__dirname, 'views') }
-
+    // Add an alt field to images, by default the title is used but
+    // we recommend using the alt one with this options for more clarity.
```
Comment (boutell) [inline app.js]: "we recommend enabling the alt field for clarity."

**Review by boutell — CHANGES_REQUESTED** [review]
Minor language tweak, then good to go. Do not merge until the feature is published in apostrophe.

File: app.js (hunk 1 above)
Comment (ValJed) [reply app.js]: Done

**Comment by ValJed** [comment]
@boutell 
Since the feature has been merged in the core I thin we can merge this one ?

**Review by boutell — APPROVED** [review]

**Pull request merged**